### PR TITLE
chore: update KendallDoesCoding profile

### PR DESCRIPTION
**Pull request by KendallDoesCoding**
## Fixes Issue
- Update profile

## Changes proposed
- Update avatar (may change it back in the future)
- Add icon in milestones section
- Add Youtube
- Color Youtube icon red.
- Remove slash from portfolio link
- Add milestone

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1067"></a>

### CODE REVIEW

**Comment by KendallDoesCoding** [comment]
@emmalearnscode @Panquesito7 Should I put my coding avatar or just my avatar for this?

**Comment by emmalearnscode** [comment]
> @emmalearnscode @Panquesito7 Should I put my coding avatar or just my avatar for my page?

What are you trying to do? Add an image to a milestone?

**Comment by KendallDoesCoding** [comment]
> > @emmalearnscode @Panquesito7 Should I put my coding avatar or just my avatar for my page?
> 
> What are you trying to do? Add an image to a milestone?

No, as the avatar for my page:
  "avatar": "https://i.ibb.co/5LsjwVB/Avatar-Maker.png",

**Comment by emmalearnscode** [comment]
We try to encourage people to use their GitHub profile link as it updates automatically here when you update GitHub and then the two are in sync.

**Comment by KendallDoesCoding** [comment]
> We try to encourage people to use their GitHub profile link as it updates automatically here when you update GitHub and then the two are in sync.

Ok, I dont really want to update my github icon in future, I added a avatar coz that looks more like a real person then this, but will update it to https://github.com/KendallDoesCoding.png

File: public/data/KendallDoesCoding.json [hunk 1]
```diff
@@ -2,13 +2,19 @@
...
-      "url": "https://kendalldoescoding.gq/",
+      "url": "https://kendalldoescoding.gq",
       "icon": "globe"
     },
+    {
+      "name": "Coding Youtube",
+      "url": "https://kendalldoescoding.gq/codingYT",
+      "icon": "youtube"
+    },
+  ]
```
Comment (vikasganiga05) [inline public/data/KendallDoesCoding.json:17]:
```suggestion

```
Remove the closing square bracket. Delete line 17.

File: public/data/KendallDoesCoding.json [hunk 2]
```diff
@@ -22,18 +28,19 @@
...
+      "icon": "link"
     },
     {
       "name": "Look at my subdomains/projects",
       "url": "https://kendalldoescoding.gq/subdomains",
       "icon": "globe"
-    }
-  ],
+    },
+  ]
```
Comment (vikasganiga05) [inline public/data/KendallDoesCoding.json:38]:
```suggestion
  ],
```
Add `,` after closing square bracket.

**Review by vikasganiga05 — CHANGES_REQUESTED** [review]
Hey, please make suggested changes to the file.

File: public/data/KendallDoesCoding.json [hunk 3]
```diff
@@ -22,18 +28,19 @@
...
   "milestones": [
     {
       "title": "Started Tech Journey",
       "date": "2020"
```
Comment (vikasganiga05) [inline public/data/KendallDoesCoding.json:42]:
```suggestion
      "date": "2020",
```
Also add `,` at the end of this line.

**Review by vikasganiga05 — APPROVED** [review]
Thanks for making the suggested changes. The JSON file looks good, but I don't have any idea why there are 2 falling checks from github-action.

**Comment by emmalearnscode** [comment]
> Thanks for making the suggested changes. The JSON file looks good, but I don't have any idea why there are 2 falling checks from github-action.

I think it’s the trailing commas on line 41 and 48 that are causing the test failures.

File: public/data/KendallDoesCoding.json [hunk 4]
```diff
@@ -22,18 +32,19 @@
...
       "url": "https://kendalldoescoding.gq/quiz",
-      "icon": "globe"
+      "icon": "link"
     },
     {
       "name": "Look at my subdomains/projects",
       "url": "https://kendalldoescoding.gq/subdomains",
       "icon": "globe"
-    }
+    },
```
Comment (emmalearnscode) [inline public/data/KendallDoesCoding.json:41]: Remove trailing comma

File: public/data/KendallDoesCoding.json [hunk 5]
```diff
@@ -22,18 +32,19 @@
...
   ],
   "milestones": [
     {
       "title": "Started Tech Journey",
-      "date": "2020"
-    }
+      "date": "2020",
+      "icon": "globe"
+    },
```
Comment (emmalearnscode) [inline public/data/KendallDoesCoding.json:48]: Remove trailing comma

**Review by emmalearnscode — CHANGES_REQUESTED** [review]

File: public/data/KendallDoesCoding.json (hunk 4 above)
Comment (KendallDoesCoding) [reply public/data/KendallDoesCoding.json]: I think this is fixed in [dc12210](https://github.com/EddieHubCommunity/LinkFree/pull/1067/commits/dc122101b7e90058974c89d6ef0ec751b00034a1) but waiting not sure

File: public/data/KendallDoesCoding.json (hunk 5 above)
Comment (KendallDoesCoding) [reply public/data/KendallDoesCoding.json:48]: I think this is fixed in [dc12210](https://github.com/EddieHubCommunity/LinkFree/pull/1067/commits/dc122101b7e90058974c89d6ef0ec751b00034a1) but waiting not sure

**Comment by vikasganiga05** [comment]
> > Thanks for making the suggested changes. The JSON file looks good, but I don't have any idea why there are 2 falling checks from github-action.
> 
> I think it’s the trailing commas on line 41 and 48 that are causing the test failures.

Yeah! that makes sense :+1:

File: public/data/KendallDoesCoding.json (hunk 5 above)
Comment (vikasganiga05) [reply public/data/KendallDoesCoding.json:48]: Hi, you forgot to add `,` on line 31.

Comment (KendallDoesCoding) [reply public/data/KendallDoesCoding.json:48]: Yup, I deleted that cause on the above review by @emmalearnscode it showed line 31 without the ',' for some reason

Comment (vikasganiga05) [reply public/data/KendallDoesCoding.json:48]: Remove the trailing comma on line 48.

**Comment by KendallDoesCoding** [comment]
Done

**Review by vikasganiga05 — APPROVED** [review]
Awesome! there are no failing checks from github-action. All are passed. Well done :+1:

**Comment by KendallDoesCoding** [comment]
> Awesome! there are no failing checks from github-action. All are passed. Well done 👍

Thanks for your and @emmalearnscode 's help :)

File: public/data/KendallDoesCoding.json [hunk 6]
```diff
@@ -2,11 +2,21 @@
...
+  "avatar": "https://github.com/KendallDoesCoding.png",
   "links": [
     {
       "name": "Check out my personal website",
-      "url": "https://kendalldoescoding.gq/",
+      "url": "https://kendalldoescoding.gq",
+      "icon": "globe"
+    },
+    {
+      "name": "Coding Youtube",
```
Comment (Panquesito7) [inline public/data/KendallDoesCoding.json:13]: I think you should capitalize the `t` 😅 What do you think?
```suggestion
      "name": "Coding YouTube",
```

**Comment by vikasganiga05** [comment]
> > Awesome! there are no failing checks from github-action. All are passed. Well done +1
> 
> Thanks for your and @emmalearnscode 's help :)

Your welcome :slightly_smiling_face: . Collaboration first code second

File: public/data/KendallDoesCoding.json (hunk 6 above)
Comment (KendallDoesCoding) [reply public/data/KendallDoesCoding.json:13]: yup lol

**Review by emmalearnscode — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Review by schmelto — APPROVED** [review]

**Review by schmelto — CHANGES_REQUESTED** [review]

File: public/data/KendallDoesCoding.json [hunk 7]
```diff
@@ -2,11 +2,22 @@
...
     {
       "name": "Check out my personal website",
-      "url": "https://kendalldoescoding.gq/",
+      "url": "https://kendalldoescoding.gq",
+      "icon": "globe"
+    },
+    {
+      "name": "Coding YouTube",
+      "url": "https://kendalldoescoding.gq/codingYT",
+      "icon": "youtube"
```
[the remark this answers is not in the document]
Comment (schmelto) [reply public/data/KendallDoesCoding.json:15]: Ups my bad just spotted this one :D

[the remark this answers is not in the document]
Comment (KendallDoesCoding) [reply public/data/KendallDoesCoding.json]: Was just gonna do this. Thanks

**Comment by KendallDoesCoding** [comment]
@Panquesito7 @emmalearnscode @vikasganiga05 Requesting a re-review from yall as I made edits after yall reviewed it before. Thanks

File: public/data/KendallDoesCoding.json [hunk 8]
```diff
@@ -33,7 +44,14 @@
...
     {
       "title": "Started Tech Journey",
-      "date": "2020"
-    }
+      "date": "2020",
+      "icon": "globe"
+    },
+    {
+      "title": "Contributed to Github Desktop",
+      "description": "I made a few small commits to the Github Desktop app."
```
Comment (schmelto) [inline public/data/KendallDoesCoding.json:52]:
```suggestion
      "description": "I made a few small commits to the Github Desktop app.",
```

Comment (KendallDoesCoding) [reply public/data/KendallDoesCoding.json]: Done in commit 
fa13d8e

File: public/data/KendallDoesCoding.json [hunk 9]
```diff
@@ -2,11 +2,22 @@
...
   "links": [
     {
       "name": "Check out my personal website",
-      "url": "https://kendalldoescoding.gq/",
+      "url": "https://kendalldoescoding.gq",
+      "icon": "globe"
+    },
+    {
+      "name": "Coding YouTube",
+      "url": "https://kendalldoescoding.gq/codingYT",
```
Comment (KendallDoesCoding) [inline public/data/KendallDoesCoding.json:14]: Should, I just put the official youtube link as if my site goes down (even for some countries) this link wont work? My site only went down 2 times, inculding the one time I messed it up, so let me know if I should keep it as it is or change it @EddieHubCommunity/linkfree .

File: public/data/KendallDoesCoding.json [hunk 10]
```diff
@@ -2,11 +2,22 @@
   "name": "Kendall Pinto",
   "type": "personal",
   "bio": "Hello, My name is Kendall. I am a Software Developer / Coder. I have made a bunch of programs in various languages, like Python, JavaScript, and the one I'm currently working on projects with, HTML & CSS. I'm currently working on the project, A Very Mogul Christmas",
-  "avatar": "https://i.ibb.co/5LsjwVB/Avatar-Maker.png",
+  "avatar": "https://github.com/KendallDoesCoding.png",
```
Comment (KendallDoesCoding) [inline public/data/KendallDoesCoding.json:5]: @EddieHubCommunity/linkfree Please let me know which avatar looks better.

https://i.ibb.co/5LsjwVB/Avatar-Maker.png
https://github.com/KendallDoesCoding.png

The first one is more of a human kinda character, than the second. The second one just has my coding name written on it. I don't mind both, but wanna know your opinion on it.

File: public/data/KendallDoesCoding.json [hunk 11]
```diff
@@ -22,7 +33,7 @@
     {
       "name": "Quiz",
       "url": "https://kendalldoescoding.gq/quiz",
-      "icon": "globe"
+      "icon": "link"
     },
     {
       "name": "Look at my subdomains/projects",
```
Comment (KendallDoesCoding) [inline public/data/KendallDoesCoding.json:39]:
```suggestion
      "name": "Check out my projects, subdomains.",
```

**Comment by emmalearnscode** [comment]
> @emmalearnscode @Panquesito7 I think we got 3 approvals, please merge after I make a final edit.

Kendall please be aware that all maintainers here are volunteers and many of us work full time jobs. We promise to get to your pull requests when we have time. Wait until you have made the final edit to post an update message, have patience and let the maintainers check things and merge things in our own time, thanks :)

**Comment by KendallDoesCoding** [comment]
> > @emmalearnscode @Panquesito7 I think we got 3 approvals, please merge after I make a final edit.
> 
> Kendall please be aware that all maintainers here are volunteers and many of us work full time jobs. We promise to get to your pull requests when we have time. Wait until you have made the final edit to post an update message, have patience and let the maintainers check things and merge things in our own time, thanks :)

Yeah, I totally understand that. Just posted a gentle reminder as this PR has been on stand-still for the last 4 days :)

**Review by krshdev — CHANGES_REQUESTED** [review]

File: public/data/KendallDoesCoding.json (hunk 10 above)
Comment (emmalearnscode) [reply public/data/KendallDoesCoding.json:5]: GitHub profile is preferable but it’s your choice.

File: public/data/KendallDoesCoding.json (hunk 9 above)
Comment (emmalearnscode) [reply public/data/KendallDoesCoding.json:14]: It’s up to you and how long you intend to pay for the domain.

File: public/data/KendallDoesCoding.json [hunk 12]
```diff
@@ -2,11 +2,22 @@
...
       "name": "Check out my personal website",
-      "url": "https://kendalldoescoding.gq/",
+      "url": "https://kendalldoescoding.gq",
+      "icon": "globe"
+    },
+    {
+      "name": "Coding YouTube",
+      "url": "https://kendalldoescoding.gq/codingYT",
+      "icon": "youtube",
+      "color": "red"
```
[the remark this answers is not in the document]
Comment (emmalearnscode) [reply public/data/KendallDoesCoding.json:16]: Either should work but test it on your dev server to make sure

**Review by emmalearnscode — COMMENTED** [review]
Looks fine as it is, just make your final decisions about the links then this should be ready to merge.

**Review by krshdev — APPROVED** [review]

**Comment by KendallDoesCoding** [comment]
> GitHub profile is preferable but it’s your choice.

Ok cool! I'll leave it has the github profile for now.

**Comment by KendallDoesCoding** [comment]
> It’s up to you and how long you intend to pay for the domain.

It's a free domain lol, I just need to renew it 😆

**Comment by KendallDoesCoding** [comment]
> > It’s up to you and how long you intend to pay for the domain.
> 
> It's a free domain lol, I just need to renew it 😆

and yeah, i do intend of renewing it.

**Review by schmelto — APPROVED** [review]

**Pull request merged**